info.puredata.Pd-extended.yaml

```diff
@@ -0,0 +1,112 @@
+app-id: info.puredata.Pd-extended
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pd-extended
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=host
```

Comment:
can't we limit this to `home` at least?

Reply:
Yeah, `home` would be a little bit better.

Reply:
I took that from net.purrdata.Purrdata, which is closely related.  I guess they determined it was needed like that.  I'm happy to consider narrowing this down, but I'd like to start with a known good configuration.